src/aeso_parser.erl

```diff
@@ -472,6 +472,19 @@ id()       -> token(id).
 tvar()     -> token(tvar).
 str()      -> token(string).
 
+binop_as_lam() ->
+    BinOps = ['&&', '||',
```

Comment:
It's sorta awkward that we maintain yet another lists of operators (one in lexer, one in expression parser and this one here)... Maybe there is a nicer way of doing so, so they won't desync?

Also, you don't want to allow `..` operator here, it doesn't make any sense to make it a lambda

Reply:
Disabled `..` here 8851629fc7f0b1d8599c6f30166b0437857d570b